workflows/ci.yml

```
@@ -0,0 +1,16 @@
+name: CI
+on: 
+  push:
+  pull_request:
+jobs:
+  nblint:
+    name: Lint Notebooks
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
```

Maybe use newer versions here?

```suggestion
      - uses: actions/checkout@v3
      - uses: actions/setup-python@v4
```